tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchAdminConsoleTests.java

```diff
@@ -21,11 +11,7 @@
 import org.alfresco.utility.data.DataContent;
 import org.alfresco.utility.data.DataSite;
 import org.alfresco.utility.data.DataUser;
-import org.alfresco.utility.model.FileModel;
-import org.alfresco.utility.model.FileType;
-import org.alfresco.utility.model.SiteModel;
-import org.alfresco.utility.model.TestGroup;
-import org.alfresco.utility.model.UserModel;
+import org.alfresco.utility.model.*;
```

Comment:
I think we try to avoid wildcard imports.

Reply:
sorry for that, really